routes/api/users.js

```diff
@@ -1,6 +1,8 @@
 const router = require("express").Router();
 const passport = require("passport");
 const User = require("../../models/").User;
+const TokenService = require('../../services/TokenService');
```

Comment:
Since `TokenService` is not a constructor, change the capital `T` to a small letter `t`.